app/views/emails/_email_form.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_for @email, url: {action: 'submit', controller: 'emails', status: params[:status]}, html: {class: 'email-form form-horizontal'} do |f| %>
+<%= form_for @email, url: @send_generic ? {action: 'submit_generic', controller: 'emails'} : {action: 'submit', controller: 'emails', status: params[:status]}, html: {class: 'email-form form-horizontal'} do |f| %>
```

Comment:
This line is pretty long, and using the ternary operator on a multi-line statement wouldn't be great either. Maybe set a variable in an if block before this line?